extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -860,31 +874,32 @@ export class AmpA4A extends AMP.BaseElement {
             result.keys = [];
           }
           return result;
+        }).then(jwkSet => {
+          return {
+            serviceName: jwkSet.serviceName,
+            keys: jwkSet.keys.map(jwk =>
+                importPublicKey(jwkSet.serviceName, jwk)
+                .catch(err => {
+                  user().error(TAG, this.element.getAttribute('type'),
+                      `error importing keys for service: ${jwkSet.serviceName}`,
+                      err, this.element);
+                  return null;
+                })),
+          };
         }).catch(err => {
           user().error(
               TAG, this.element.getAttribute('type'), err, this.element);
-          return {serviceName: currServiceName};
+          return {serviceName: currServiceName, keys: []};
```

Comment:
Given we are fetching these before we need them, let's add a TODO here that we could re-attempt to fetch the keys against if it fails